test/gst-va/encode/vp9.py

```diff
@@ -0,0 +1,170 @@
+###
+### Copyright (C) 2022-2023 Intel Corporation
+###
+### SPDX-License-Identifier: BSD-3-Clause
+###
+
+from ....lib import *
+from ....lib.gstreamer.va.util import *
+from ....lib.gstreamer.va.encoder import EncoderTest
+
+spec = load_test_spec("vp9", "encode", "8bit")
+
+@slash.requires(*have_gst_element("vavp9dec"))
+class VP9EncoderBaseTest(EncoderTest):
+  def before(self):
+    super().before()
+    vars(self).update(
+      codec         = "vp9",
+      gstdecoder    = "vavp9dec",
+      gstmediatype  = "video/x-vp9",
+      gstparser     = "vp9parse",
+      gstmuxer      = "matroskamux",
+      gstdemuxer    = "matroskademux",
+    )
+
+  def get_file_ext(self):
+    return "webm"
+
+@slash.requires(*have_gst_element("vavp9lpenc"))
```

Comment:
This is the wrong encoder requirement for non-lp

Reply:
Thank you for your review, I have made corresponding modifications to it